gammapy/background/energy_offset_array.py

```diff
@@ -104,3 +127,76 @@ def plot_image(self, ax=None, offset=None, energy=None, **kwargs):
         image = ax.imshow(self.data, extent=extent, **kwargs)
         plt.colorbar(image)
         return ax
+
+    @classmethod
+    def read(cls, filename):
+        """Create `EnergyOffsetArray` from  fits file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdu_list = fits.open(filename)
+        return cls.from_fits(hdu_list)
+
+    
+    @classmethod
+    def from_fits(cls, hdu, scheme=None):
+        """Read EnergyOffsetArray from a fits binary table.
+
+        Parameters
+        ----------
+        hdu : `~astropy.io.fits.BinTableHDU`
+            HDU binary table for the EnergyOffsetArray.
+        Returns
+        -------
+        cube : `~gammapy.background.EnergyOffsetArray`
+            EnergyOffsetArray object.
+        """
+
+        header = hdu[1].header
+        data = hdu[1].data
+        
+        # get offset and energy binning
+        offset_edges = _make_bin_edges_array(data['THETA_LO'], data['THETA_HI'])
+        energy_edges = _make_bin_edges_array(data['ENERG_LO'], data['ENERG_HI'])
+        
+        # get data
+        energy_offset_array=data['EnergyOffsetArray'][0]
```

Comment:
I think this [0] is needed with the way the array is stored in FITS (as one row).
In other parts of the code we called [.squeeze](http://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.squeeze.html) instead:
https://github.com/gammapy/gammapy/blob/79502a0a525088173cee37576018b6e46097141e/gammapy/irf/energy_dispersion.py#L531
Can you please also do this here?

(we should re-factor the whole gammapy.irf FITS I/O code in the coming weeks to abstract that functionality to handle arrays and axes into a base class or utility functions ... for now it's repeated in every IRF class)